File: src/main/java/app17/rest/RideRequest.java
package app17.rest;


public class RideRequest {
    String id = null;
    Number startingLocation_Lat, startingLocation_Lon, endingLocation_Lat, endingLocation_Lon;

    public RideRequest(Number startingLocation_Lat, Number startingLocation_Lon, Number endingLocation_Lat,
                       Number endingLocation_Lon) {
        this.startingLocation_Lat = startingLocation_Lat;
        this.startingLocation_Lon = startingLocation_Lon;
        this.endingLocation_Lat = endingLocation_Lat;
        this.endingLocation_Lon = endingLocation_Lon;
    }
    public void setId(String id) {
        this.id = id;
    }
}

File: src/resources/rideRequest.js
db = connect("localhost:27017/app17");
coll = db.rideRequest;
coll.insert({
    "startingLocation_Lat" : Number(41.13452345),
    "startingLocation_Lon" : Number(44.13452345),
    "endingLocation_Lat" : Number(44.13452345),
    "endingLocation_Lon" : Number(52.13452345)});
